internal/backends/qt/qt_window.rs

```diff
@@ -1702,6 +1698,18 @@ impl WindowAdapter for QtWindow {
         }};
     }
 
+    fn set_fullscreen(&self, fullscreen: bool) {
+        let widget_ptr = self.widget_ptr();
+        cpp! {unsafe [widget_ptr as "QWidget*", fullscreen as "bool"] {
+                if (fullscreen) {
+                    widget_ptr->setWindowState(Qt::WindowFullScreen);
+                } else {
+                    widget_ptr->setWindowState(Qt::WindowNoState);
+                }
+            }
```

Comment:
This would also clear other states like minimized and maximized.
In fact, the best would be to add or remove the Qt::WindowFullScreen flag from the set of state using windowStates and setWindowStates

Reply:
That makes a lot of sense, done.